cmd/helm/install.go

```diff
@@ -215,16 +215,18 @@ func runInstall(args []string, client *action.Install, valueOpts *values.Options
 		// https://github.com/helm/helm/issues/2209
 		if err := action.CheckDependencies(chartRequested, req); err != nil {
 			if client.DependencyUpdate {
-				man := &downloader.Manager{
-					Out:              out,
-					ChartPath:        cp,
-					Keyring:          client.ChartPathOptions.Keyring,
-					SkipUpdate:       false,
-					Getters:          p,
-					RepositoryConfig: settings.RepositoryConfig,
-					RepositoryCache:  settings.RepositoryCache,
-					Debug:            settings.Debug,
-				}
+				man := downloader.NewManager(
+					out,
+					cp,
+					downloader.VerifyNever,
```

Comment:
Why should a dependency update never verify the signing credentials? Shouldn't this be an opt-in based on whether the user specified the `--verify` flag?

https://github.com/helm/helm/blob/d55c53df4e394fb62b0514a09c57bce235dd7877/cmd/helm/flags.go#L51

Reply:
I don't believe I'm changing any existing behavior here, these arguments are just explicit instead of implicit.
You can see some of the other functions have checks for the verify flag:
```go
			verify := downloader.VerifyNever
			if client.Verify {
				verify = downloader.VerifyAlways
			}
```
But this one didn't